### PR TITLE
Update build.rs

### PR DESCRIPTION
**Pull request by 0xdeafbeef**

### CODE REVIEW

**Review by nesquikm — CHANGES_REQUESTED** [review]
@0xdeafbeef thnx :)

**Comment by Alex-A4** [comment]
Why we need this changes?

**Comment by nesquikm** [comment]
> Why we need this changes?

I think in order to verbose report errors and create paths more correctly (CARGO_MANIFEST_DIR). Am I right? :)

File: packages/nekoton_bridge/native/build.rs [hunk 1]
```diff
@@ -2,13 +2,27 @@ use lib_flutter_rust_bridge_codegen::{
...
-const RUST_INPUT: &str = "src/merged.rs";
 const DART_OUTPUT: &str = "../lib/src/bridge_generated.dart";
 const IOS_C_OUTPUT: &str = "../../flutter_nekoton_bridge/ios/Classes/frb.h";
 // const MACOS_C_OUTPUT: &str = "../../flutter_nekoton_bridge/macos/Classes/frb.h";
 
 fn main() {
+    let manifest_dir = PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap());
+
+    let rust_input = manifest_dir
+        .join("src/merged.rs")
```
[the remark this answers is not in the document]
Comment (nesquikm) [reply packages/nekoton_bridge/native/build.rs:15]: @0xdeafbeef ping?

**Pull request closed without merging**